app/uk/gov/hmrc/merchandiseinbaggage/views/ViewUtils.scala

```diff
@@ -41,4 +41,23 @@ object ViewUtils {
 
   lazy val exportCountries: List[Country] =
     CountryService.getAllCountries.filterNot(_.code == "GB")
+
+  def proofOfOriginNeeded(declaration: Declaration): Boolean = {
```

Comment:
we must already have this logic somewhere in `utils`, please check

Reply:
done